packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -714,7 +694,7 @@ abstract class BaseFlutterTask extends DefaultTask {
     String localEngineSrcPath
     @Input
     String targetPath
-    @Optional @Input
+    @Optional
```

Comment:
changing tool verbosity shouldn't force the task to rerun.

Reply:
It might be possible that someone wants to re-run the command with the `-v` flag the same way it ran without the `-v` flag.